assets/js/googlesitekit/datastore/user/permissions.js

```diff
@@ -62,6 +71,21 @@ export const actions = {
 			type: SET_PERMISSION_SCOPE_ERROR,
 		};
 	},
+
+	/**
+	 * Sets user capabilities.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} capabilities User capabilities.
+	 * @return {Object} Redux-style action.
+	 */
+	setCapabilities( capabilities ) {
```

Comment:
This should be called `receiveCapabilities` per our naming conventions, and marked as `@private`. We should only use `set*` for actions which we intend to have publicly used to set something, but this one here only exists to set data that comes from the server / API, it should never be called directly, except in our tests possibly.

For those types of actions that "receive" data from the server, let's always use `receive*` (and also `RECEIVE_*` for reducer etc).

Reply:
Updated.